src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/nomis/Mapping.kt

```diff
@@ -20,12 +20,12 @@ fun NomisReport.toNewEntity(): Report {
     title = title ?: "NO DETAILS GIVEN",
     description = description ?: "NO DETAILS GIVEN",
     reportedBy = reportingStaff.username,
-    reportedDate = reportedDateTime,
+    reportedAt = reportedDateTime,
```

Comment:
note that the field names on the right are not changed as these are coming from NOMIS and those DTOs are not changing.